tests/unit/helpers/validate-properties-fail-test.js

```diff
@@ -0,0 +1,61 @@
+import { moduleFor } from 'ember-qunit';
```

Comment:
Agreed, naming is hard. `-fail` was just a place holder while we got it working, now we should make the name better :smile: